views/atom.blade.php

```diff
@@ -13,10 +13,9 @@
     <entry>
         <title><![CDATA[{!! $entry['title'] !!}]]></title>
         <link rel="alternate" type="text/html" href="{{ $entry['permalink'] }}"/>
-        <id>{{ $entry['id'] or $entry['permalink'] }}</id>
+        <id>{{ $entry['permalink'] }}</id>
         <updated>{{ $entry['pubdate']->format(DateTime::ATOM) }}</updated>
         <summary><![CDATA[{!! $entry['description'] !!}]]></summary>
```

Comment:
As posts can include HTML, we should add an attribute here for the feed to validate.
```suggestion
        <summary type="html"><![CDATA[{!! $entry['description'] !!}]]></summary>
```

Reply:
The HTML is not included if you stick with `$entry['description']`. It seems like the HTML is only included if you use `$entry['content']`. Which is currently in your code. I removed it in my fork because I do not want the HTML. I only want a plain text feed. You may test my feed which uses my forked version of this code by plugging the following address into a feed validator: https://ennuily.com

Some posts at this forum have HTML. But, you will see no HTML in my feed. 

I guess this is a personal preference. Some may want HTML in their feeds. I do not which is why I went with the description only versus the content.

Reply:
I guess too that's a personal preference. To be honest, I'm on the « full+html feed » to be able to consume the whole content from an aggregator without having to open the forum. But I understand that's not the prefered behaviour of everyone. I could for the moment use your solution (so I don't add even more time to this upgrade) and later add an option between text and HTML; and between full content and excerpts.

Reply:
Yes, the ideal solution would be admin settings. This way people can switch from full content with HTML to just text only excerpts. You could have 4 blade.php files. Two RSS and two Atom. The admin setting would switch between these depending on the HTML/non-HTML preference. 

It’s your extension so if your preference is full content / full HTML I say go for it. 🙂 You could accept and merge this pull request which fixes a couple things, and then change the “description” code to “content” and add that HTML attribute where needed.

Another option - no admin settings needed... copy the RSS and Atom blade files and give them a different name. By default the extension could use the full HTML versions. But, in the install instructions you tell people that they can simply rename the non-HTML versions of the files to the appropriate names, and either delete or rename the full HTML files. It’s manual, but gives people a way to somewhat easily switch between the two options.